JoseLuisOnofreFranco/practica6/preprocessing.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+def preprocess_corpus(corpus: list[str]) -> list[list[str]]:
+    cleaned = [ delete_scape_sequences(sentence) for sentence in corpus]
+    cleaned = [ sentence for sentence in cleaned if not is_empty_sentence(sentence)]
+    cleaned = [ preprocess_sentence(sentence) for sentence in cleaned]
+    cleaned = [ delete_empty_spaces(sentence) for sentence in cleaned]
+
+    return [ sentence.split() for sentence in cleaned ]
+
```

Comment:
Veo algunos espacios de más entre corchetes

```suggestion
def preprocess_corpus(corpus: list[str]) -> list[list[str]]:
    cleaned = [delete_scape_sequences(sentence) for sentence in corpus]
    cleaned = [sentence for sentence in cleaned if not is_empty_sentence(sentence)]
    cleaned = [preprocess_sentence(sentence) for sentence in cleaned]
    cleaned = [delete_empty_spaces(sentence) for sentence in cleaned]

    return [sentence.split() for sentence in cleaned ]

```